priv/configs/redis_sentinel.conf

```diff
@@ -0,0 +1,10 @@
+port 26379
+daemonize yes
+sentinel myid 10022d3099331ec79bd29f9179c82be7bec80441
+pidfile "/var/run/redis.pid"
+user default on nopass ~* +@all
```

Comment:
The CI job seems to fail on this line for Redis 4-5. I don't think there was ACL in these Redis versions.